nselib/http.lua

```diff
@@ -87,6 +87,12 @@
 --   end
 --   </code>
 -- * <code>no_cookie_overwrite</code>: Do not overwrite the cookies initially provided and when a duplicate cookie is received, ignore it.
+-- * <code>enable_cookies</code>: Cookies are accessible to other http calls
+-- The following example code demonstrates how the cookies can be passed and updated for every subsequent http calls.
+--
+-- <code> 
+-- http.get(host, port, {no_cookie_overwrite=false, cookies = options.cookies, enable_cookie = true})
```

Comment:
Exactly. You need to store the cookie jar somewhere in order to be used in subsequent requests. Think about how users will use the library. Implementing cookie support in scripts will give you a better idea on what the best workflow for this.